vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-integration-tests/src/test/java/com/vaadin/flow/component/spreadsheet/test/CustomEditorIT.java

```diff
@@ -354,6 +354,91 @@ public void customEditorIsVisible_sheetIsChanged_editorsRemoved() {
         }
     }
 
+    @Test
```

Comment:
Both the tests seem to pass without the fixes.

Reply:
I have removed one of the tests and refactored the other one to make it so that it breaks in `main`.